certified-connectors/AutoPylot Connect/Readme.md

```diff
@@ -0,0 +1,39 @@
+# AutoPylot Connect
+
```

Comment:
Please update readme.md file as per the template here - https://github.com/microsoft/PowerPlatformConnectors/blob/3ac7cb2ca3a494950d3a6810d2aeeb69f7ccc507/templates/certified-connectors/apiProperties.json

Reply:
Updated according to template https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md